### DISCUSSION TITLE
[#991909] UI resets when changing dimensions

### DISCUSSION THREAD

**Discussion by tryashtar** [Mojang/minecraft-editor#33] [Bugs]
How to reproduce
1. Open the paste menu
2. Change dimensions from the overworld to the nether
3. Wait

Expected result: paste menu is still open
Actual result: it's gone, selection menu is now open instead

**Comment by minecraftchloe** [comment]
Repro'd and duplicate to https://github.com/Mojang/minecraft-editor/discussions/11

**Comment by minecraftchloe** [comment]
Should be fixed! Let us know if this is an issue again!
